src/overlays/actors/ovl_En_Fishing/z_en_fishing.c

```diff
@@ -5419,9 +5419,8 @@ void EnFishing_UpdateOwner(Actor* thisx, PlayState* play2) {
                         mainCam->eyeNext = sSubCamEye;
                         mainCam->at = sSubCamAt;
                         func_80169AFC(play, sSubCamId, 0);
-                        Cutscene_End(play, &play->csCtx);
-                        func_800B7298(play, &this->actor,
-                                      PLAYER_CSMODE_6); // arg2 changed from PLAYER_CSMODE_7 to PLAYER_CSMODE_6 in MM
```

Comment:
Indeed both are `PLAYER_CSMODE_END` and the enum just got shifted from OoT to MM. Not sure if it's worth the comment since it's named in MM and will likely sync with OoT's name as well

Reply:
Yeah that makes perfect sense, thanks! In that case, yeah, the comment is fine to be deleted, like you already did. Just wanted to make sure we weren't losing any documentation.